### DISCUSSION TITLE
gitbook and slack

### DISCUSSION THREAD

**Discussion by dusohee** [GitbookIO/community#262] [Feature Requests]
When gitbook and slack connected, new post was noticed like below. 
![20230707_164214](https://github.com/GitbookIO/community/assets/98582132/2c5e95ea-c8ab-4a18-8616-4cf7ef45a78c)

It will be better to show preview img and summary of new post like other websites. 
![20230707_164221](https://github.com/GitbookIO/community/assets/98582132/a8c99dd5-2bc6-425d-a8f1-b92a98cd0307)

**Comment by taszap** [comment]
Closing this in favour of #103
